evergreen.yml

```
@@ -1960,7 +2052,44 @@ tasks:
   - name: semgrep
     exec_timeout_secs: 3600 # 1h
     commands:
-      - func: "static code analysis"
+      - func: "generate static code analysis"
+
+  - name: ssdlc-artifacts-release
+    run_on: ubuntu2204-small
+    git_tag_only: true
+    depends_on:
+      - name: "release"
+        variant: "release"
+      - name: sbom
+        variant: code-quality-security
+      - name: semgrep
+        variant: code-quality-security
+      - name: compliance-report
+        variant: code-quality-security
+    exec_timeout_secs: 300 # 5m
+    commands:
+      - func: "publish static code analysis"
+      - func: "publish compliance report"
+      - func: "pull augmented SBOM from Silk"
+      - func: "publish augmented SBOM"
```

The functions are sequential, if the static code analysis or augmented SBOM publication fails, the compliance report should not be created. It will point to non-existent file(s).
Also, call the `generate compliance report` function directly. Unless there is a reason that I didn't see for which you used a task.
```suggestion
    run_on: ubuntu2204-small
    git_tag_only: true
    depends_on:
      - name: "release"
        variant: "release"
      - name: sbom
        variant: code-quality-security
      - name: semgrep
        variant: code-quality-security
    exec_timeout_secs: 300 # 5m
    commands:
      - func: "publish static code analysis"
      - func: "pull augmented SBOM from Silk"
      - func: "publish augmented SBOM"
      - func: "generate compliance report"
      - func: "publish compliance report"
```